fsspec/tests/conftest.py

```diff
@@ -185,4 +197,5 @@ def serve():
 @pytest.fixture(scope="module")
 def server():
     with serve() as s:
-        yield s
+        server = SimpleNamespace(address=s, realfile=_make_realfile(s))
```

Comment:
named_tuple or dict? I haven't really seen SimpleNamespace used much. In effect, it's a dictionary here. May as well include server name and other strings that are templated throughout the tests.

Reply:
I thought that might be a bit overkill for tests.

Are there really any other strings that could be put in here? I replaced all the `realfile`s which seemed like the only common ones.

Reply:
Yeah, OK, maybe you're right :)